airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -336,6 +336,8 @@ def _read_pages(
         # Always return an empty generator just in case no records were ever yielded
         yield from []
 
+    _read_single_page = _read_pages
```

Comment:
This solution should not be needed after https://github.com/airbytehq/airbyte/pull/45601 is merged. I'll work on that today

Reply:
Got it, I'll remove this line afterward​s.

Reply:
The fix is [here](https://github.com/airbytehq/airbyte/pull/45626/files#diff-0572edeedd561901f8110c0f88b48efde87d1ed91247c0eb189c7229da12090dR113-R123)